etstool.py

```diff
@@ -103,6 +103,7 @@
     "traitsui",
     "configobj",
     "coverage",
+    "importlib_resources",
```

Comment:
Does this have a version range requirement `importlib_resources >= 1.1.0`?

This is now a test dependency, but really, apptools has plenty of test dependencies that are not runtime dependencies and apptools has not been maintaining a separate set of dependencies in `setup.py`, but I think it should.  Ideally, we should be able to run the test suite using `pip`:
```
pip install apptools[test]
python -m unittest discover apptools
```

What do you think about adding a test dependency group in setup.py as part of this PR? (Just the `importlib_resources` one would be good enough for now).

Reply:
thanks for pointing that out. you're right, we need v `>=1.1.0`. I'll add a test dependency group.